### PR TITLE
Resync all files for fixing SPI message ID

### PR DESCRIPTION
**Pull request by benjamin-confino**
This resyncs all the files involved in updating the CDI SPI message ID and delivers them in one place.

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops [hunk 1]
```diff
@@ -69,6 +69,6 @@ implicit.bean.scanning.disabled.CWOWB1009W=CWOWB1009W: Implicit bean archives ar
 implicit.bean.scanning.disabled.CWOWB1009W.explanation=Archives are not scanned for CDI beans unless they have a beans.xml file.
 implicit.bean.scanning.disabled.CWOWB1009W.useraction=Ensure that every archive that uses CDI has a beans.xml file or re-enable implicit beans scanning in your server.xml file.
 
-spi.extension.failed.to.construct.CWOWB10010E=CWOWB10010E: Failed to create CDI Extension via SPI.
-spi.extension.failed.to.construct.CWOWB10010E.explanation=CDI failed to invoke the constructor of the extension class {0} registered via the getExtensions() method of a CDIExtensionMetaData implementation because of the exception {1}.
-spi.extension.failed.to.construct.CWOWB10010E.useraction=Fix the error preventing your extension from instantiating.
+spi.extension.failed.to.construct.CWOWB1010E=CWOWB1010E: Failed to create CDI Extension via SPI.
+spi.extension.failed.to.construct.CWOWB1010E.explanation=CDI failed to invoke the constructor of the extension class {0} that is registered via the getExtensions() method of a CDIExtensionMetadata implementation because of the exception {1}.
```
Comment (helyarp) [inline dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops:73]: Minor change suggested: Change "via" to "with"

Comment (benjamin-confino) [reply dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops:73]: Sure

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Pull request merged**